hail/python/test/hail/methods/test_impex.py

```diff
@@ -1223,6 +1223,15 @@ def test_import_locus_intervals_no_reference_specified(self):
         self.assertTrue(t.count() == 2)
         self.assertEqual(t.interval.dtype.point_type, hl.tstruct(contig=hl.tstr, position=hl.tint32))
 
+    def test_import_locus_intervals_recoding(self):
+        interval_file = resource('annotinterall.grch38.no.chr.interval_list')
+        t = hl.import_locus_intervals(interval_file,
+                                      contig_recoding={str(i): f'chr{i}' for i in [*range(1, 23), 'X', 'Y', 'M']},
+                                      reference_genome='GRCh38')
+        t._force_count()
+        self.assertTrue(t.count() == 3)
```

Comment:
You can combine these two lines as you did below.

Reply:
also, don't use assertTrue. use assertEqual, but standard python `assert t.count() == 3` is preferred with pytest

Reply:
I removed every occurrence of `assertTrue(... == ...)` from this file.